FirebasePerformance/Sources/AppActivity/FPRSessionDetails.m

```diff
@@ -36,12 +36,13 @@ - (instancetype)initWithSessionId:(NSString *)sessionId options:(FPRSessionOptio
 - (FPRSessionDetails *)copyWithZone:(NSZone *)zone {
   FPRSessionDetails *detailsCopy = [[[self class] allocWithZone:zone] initWithSessionId:_sessionId
                                                                                 options:_options];
+  detailsCopy.sessionCreationTime = _sessionCreationTime;
```

Comment:
Is this change about another bug fix?

Reply:
This is the critical change that lead to the issue. Everywhere in other traces, we take a copy of the sessionDetails, but the session creation time is not copied over. That could lead to this possible crash.

Reply:
Sg! Thanks!